README.md

```diff
@@ -6,64 +6,10 @@
 [![pyversions](https://img.shields.io/pypi/pyversions/nucypher.svg)](https://pypi.org/project/nucypher/)
 [![codecov](https://codecov.io/gh/nucypher/nucypher/branch/development/graph/badge.svg?token=9w1dAFqog5)](https://codecov.io/gh/nucypher/nucypher)
 [![discord](https://img.shields.io/discord/411401661714792449.svg?logo=discord)](https://discord.gg/7rmXa3S)
-[![Documentation Status](https://readthedocs.org/projects/nucypher/badge/?version=latest)](https://nucypher.readthedocs.io/en/latest/)
 [![license](https://img.shields.io/pypi/l/nucypher.svg)](https://www.gnu.org/licenses/gpl-3.0.html)
 
 ----
 
-The NuCypher network provides accessible, intuitive, and extensible runtimes and interfaces
```

Comment:
If removing this then you might as well remove the rest of the README i.e. Whitepapers, Network, Economics, Cryptography.

That being said instead of leaving it empty you can either link to the Threshold Docs eg. https://docs.threshold.network/applications/threshold-access-control, AND/OR copy the same blurb from the link.

My preference would be to copy the same blurb form the link here, and link to the docs for "more information" or something like that.

Reply:
✔️